LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/navigation/root-element-transition-iframe-with-startVT-on-main.html

```diff
@@ -29,6 +29,9 @@
   ::view-transition-new(inner) {
     opacity: 1;
   }
+  ::view-transition-group(inner) {
+    animation-duration: 0s;
```

Comment:
`animation-play-state: paused` ?

Reply:
I think we'd need to set it to the end too?

Reply:
Oh I guess maybe the play-state wouldn't inherit. Whatever works and isn't flaky seems fine to me.